Apps/Examples/Examples/All Examples/UpdatePointAnnotationPositionExample.swift

```diff
@@ -40,7 +40,9 @@ public class UpdatePointAnnotationPositionExample: UIViewController, ExampleProt
     }
 
     @objc public func updatePosition(_ sender: UITapGestureRecognizer) {
-        let newCoordinate = mapView.coordinate(for: sender.location(in: mapView))
+        let point = mapView.convert(sender.location(in: mapView), to: mapView)
+        let screenCoordinate = ScreenCoordinate(x: Double(point.x), y: Double(point.y))
+        let newCoordinate = try! mapView.__map.coordinateForPixel(forPixel: screenCoordinate)
```

Comment:
Should we hold off on this change until we can introduce a replacement for `mapView.coordinate(for:)`?

Reply:
@julianrex @nishant-karajgikar What are your thoughts?

Reply:
just talked to Julian, we are going to revert this for now